query/src/test/java/org/infinispan/query/test/Book.java

```diff
@@ -0,0 +1,49 @@
+package org.infinispan.query.test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.hibernate.search.annotations.Field;
+import org.hibernate.search.annotations.Indexed;
+import org.hibernate.search.annotations.IndexedEmbedded;
+
+@Indexed
+public class Book {
```

Comment:
should this file contain equals and hashcode as well ?

Reply:
Done